### DISCUSSION TITLE
Farming pp in laazer feels unfair

### DISCUSSION THREAD

**Discussion by Asiskan** [ppy/osu#28104] [Ideas]
Don't get me wrong i think the slider accuracy changes are genuinely great, but as of right now , it feels like there is no point in booting up lazer if the play session's goal is to farm and try to rank up, slider accuracy adds a new layer of difficulty that is often easiest to just ignore and stick to stable, acccuracy to pp being the same in lazer and stable feels kind of unfair when stable is marginally easier to acc in.

**Comment by kaysting** [comment]
I agree. I play casually on lazer and don't really farm anymore, but I have set some fluke scores. I don't see people seriously trying to farm on lazer until stable gets the classic 0.96 score multiplier. That alone will drive people to switch, especially if classic plays become worth less pp or lazer non-classic plays grant more pp (which I think should be the case).

**Reply by bdach** [reply]
> I don't see people seriously trying to farm on lazer until stable gets the classic 0.96 score multiplier

I'm not sure what that has to do with "farming". Score doesn't have to do anything with earning pp anymore. Total pp processing has been changed, the score that counts towards your pp on a map is your highest *pp* score on that map, not the score with the most *score* on that map. Or, in other words, "score-V1'ing" is no longer a thing.

**Comment by dani211e** [comment]
> acccuracy to pp being the same in lazer and stable...

FYI there is already a [PR open](https://github.com/ppy/osu/pull/27063) to deal with this issue. From a bit of testing with my own top plays running them with + without CL led to 3%-10% increase in pp. 
It's true that currently they are worth the same, but whenever this gets merged and a recalc is deployed, all your submitted ranked scores w/ slideracc will be updated as well, so you might as well start now.

**Reply by Givikap120** [reply]
yes, just pp system is not considering for lazer gameplay changes yet (not only slider acc btw)
it's a temporal issue
